cli/api_claim_command.go

```
@@ -0,0 +1,71 @@
+
+		params := ClaimParams{
+			ClaimToken: args[0],
+			Hostname:   cmd.String("hostname"),
+			Aliases:    cmd.StringSlice("alias"),
+			Comment:    cmd.String("comment"),
+			Emoji:      cmd.String("emoji"),
+		}
+
+		resp, err := reqClaim(params)
+		if err != nil {
+			printError(err.Error())
```

golangci-lint (1.62.2)

printf: non-constant format string in call to github.com/husarnet/husarnet/cli/v2.printError

(govet)